### PR TITLE
Dev 008 creating login frontend

### PR DESCRIPTION
**Pull request by Paul-Gerarts**

### CODE REVIEW

File: src/main/java/be/syntra/devshop/DevshopFront/controllers/AuthorisationController.java [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+        model.addAttribute("user", new RegisterUserDto());
+        return "/user/register";
+    }
+
+    @PostMapping("/register")
+    public Object getRegisterFormEntry(@ModelAttribute("registerForm")RegisterUserDto registerUserDto, Model model){
+        StatusNotification statusNotification = authorisationService.register(registerUserDto);
+        model.addAttribute("user", registerUserDto);
+        model.addAttribute("status", statusNotification);
+        return !statusNotification.equals(StatusNotification.SUCCES)
```
Comment (lenshuygh) [inline src/main/java/be/syntra/devshop/DevshopFront/controllers/AuthorisationController.java]: isn't it slightly more readable if you put the whole condition between () ?

File: src/main/java/be/syntra/devshop/DevshopFront/security/CustomLogoutHandler.java [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+import org.springframework.security.core.Authentication;
+import org.springframework.security.web.authentication.logout.LogoutSuccessHandler;
+
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.io.IOException;
+
+public class CustomLogoutHandler implements LogoutSuccessHandler {
+
+    private Logger logger = LoggerFactory.getLogger(CustomLogoutHandler.class);
```
Comment (lenshuygh) [inline src/main/java/be/syntra/devshop/DevshopFront/security/CustomLogoutHandler.java]: I tried logging with annotated  @Slf4j in other parts and it should work

File: src/main/java/be/syntra/devshop/DevshopFront/controllers/AuthorisationController.java (hunk 1 above)
Comment (Paul-Gerarts) [reply src/main/java/be/syntra/devshop/DevshopFront/controllers/AuthorisationController.java]: semantics. Changed to your preference :p

File: src/main/java/be/syntra/devshop/DevshopFront/security/CustomLogoutHandler.java (hunk 2 above)
Comment (Paul-Gerarts) [reply src/main/java/be/syntra/devshop/DevshopFront/security/CustomLogoutHandler.java]: good to know. Won't refactor untill fixing tests

**Pull request merged**